static/controller/daemonset.yml

```diff
@@ -0,0 +1,50 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: metallb
+    component: controller
+  name: controller
+  namespace: metallb-system
```

Comment:
Using now a filter for the Helm Chart. The namespace object is created manually, because there is an additional namespace label and there seems not be a way to provide that.